Apuleius/opensource/apuleius.ga_lat.xml.json

```diff
@@ -1 +0,0 @@
ilestone": {"@unit": "chapter", "@n": "3"}, "#text": "Diu denique deliberaverat secum Thrasyllus, quod nec clandestinis colloquiis opportunum repperiret locum, et adulterinae Veneris magis magisque praeclusos aditus copia custodientium cerneret novaeque atque gliscentis affectionis firmissimum vinculum noni posse dissociari perspiceret, et puellae, si vellet, quanquam velle non posset, furatrinae coniugalis incommodaret rudimentum; et tamen ad hoc ipsum quod non potest, contentiosa pernicie, quasi posset, impellitur: quod nunc arduum factu putatur, amore per dies roborato facile videtur effectu. Spectata denique sed, oro, sollicitis animis intendite, quorsum furiosae libidinis proruperint impetus."}, {"milestone": [{"@unit": "chapter", "@n": "4"}, {"@unit": "chapter", "@n": "5"}, {"@unit": "chapter", "@n": "6"}], "pb": [{"@id": "p.350"}, {"@id": "p.352"}], "q": "Quid stupore confusi vel etiam cassa formidine similes humilitati servorum istorum, vel in modum pavoris feminei deiecti tam opimam praedam mediis manibus amittimus? Quin equos inscendimus? Quin ocius indipiscimur? En cape venabulum, ei ego sumo lanceam", "#text": "Die quadam venatum Tlepolemus assumpto Thrasyllo petebat indagaturus feras, si quid tamen in capreis feritatis est; nec enim Charite maritum suum quaerere patiebatur bestias armatas dente vel cornu. Iamque apud frondosum tumulum ramorumque densis tegminibus umbrosum, prospectu vestigatorum obsaeptis capreis, canes venationis indagini generosae, mandato cubili residentes invaderent bestias, immittuntur; statimque sollertis disciplinae memores, partitae totos praecingunt aditus tacitaque prius servata mussitatione, signo sibi repentino reddito, latratibus fervidis dissonisque miscent omnia. Nec ulla caprea nec pavens damula nec prae ceteris\n\n\nferis mitior cerva, sed aper immanis atque invisitatus exsurgit toris callosae cutis obesus, pilis inhorrentibus corio squalidus, setis insurgentibus spinae hispidus dentibus attritu sonaci spumeus, oculis aspectu minaci flammeus, impetu saevo frementis oris totus fulmineus; et primum quidem canum procaciores quae comminus contulerant vestigium, genis hac illac iactatis consectas interficit, dein calcata retiola,  qua primos impetus reduxerat, transabiit. Et nos quidem cuncti pavore deterriti et alioquin innoxiis venationibus consueti, tunc etiam inermes atque immuniti, tegumentis frondis vel arboribus latenter abscondimus; Thrasyllus vero nanctus fraudium opportunum decipulum sic Tlepolemum captiose compellat: ; nec tantillum morati protinus insiliunt equos ex summo studio bestiam insequentes. Nec tamen illa genuini vigoris oblita retorquet imi petum et incendio feritatis ardescens dente compulsa quem primum insiliat cunctabunda rimatur. Sed prior Tlepolemus iaculum, quod gerebat, insuper dorsum bestiae contorsit: at Thrasyllus ferae quidem pepercit sed equi, quo vehebatur Tlepolemus, postremos poplites lancea feriens amputat. Quadrupes reccidens, qua sanguis effluxerat, toto tergo supin
```

Comment:
@kigawas In the GH PR, it reads "This file was deleted." -- was this intentional? There are >700 file changes, so it's hard for me to see what all has been altered. Thx

Reply:
You can use file filter to hide .json and .xml files